operators/labInstance-operator/README.md

```diff
@@ -35,17 +37,58 @@ kubectl create configmap kubevirt-config -n kubevirt --from-literal debug.useEmu
 kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-cr.yaml
 ```
 
-### Development
+### Install the CRDs
+
+Before the deploying the operator, we have to add the LabInstance and LabTemplate CRDs. This can be done via the Kubebuilder-provided Makefile:
+
+```bash
+make install
+make install-lab-template
+```
+
+or directly via the commands:
+
+```bash
+kubectl kustomize config/crd | kubectl apply -f -
+kubectl kustomize config/crd | kubectl delete -f -
+```
+
+### Deployment
+To install CrownLabs in your cluster, you have to do the following steps.
```

Comment:
I would say something more specific than "install CrownLabs", since this is not a guide on how to install CrownLabs.